Hunting Queries/AzureDiagnostics/CriticalPortsOpened.yaml

```diff
@@ -38,4 +38,4 @@ query: |
   | where type_s !~ "block"
   | order by TimeGenerated desc
   | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), count() by OperationName, systemId_g, vnetResourceGuid_g, subnetPrefix_s, macAddress_s, primaryIPv4Address_s, ruleName_s,
-direction_s, priority_d, type_s, conditions_destinationIP_s, conditions_destinationPortRange_s, conditions_sourceIP_s, conditions_sourcePortRange_s, ResourceId
+   direction_s, priority_d, type_s, conditions_destinationIP_s, conditions_destinationPortRange_s, conditions_sourceIP_s, conditions_sourcePortRange_s, ResourceId
```

Comment:
Not sure anything changed here?

Reply:
indentation added in project on last line. no other change